File: AHome/30.01/XO/HenesseyXO/HenesseyXO/Sessions/TTTpacket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HenesseyXO.Sessions
{
    public class TTTpacket
    {
        public PlayerTurn Turn { set; get; }
        public string Unit { set; get; }
        public int ButtonNumber { set; get; }
        public string[] Matrix { set; get; }
        public string GameResult { set; get; }

        public TTTpacket(PlayerTurn turn, string unit, int buttonNumber, string[] matrix, string gameResult)
        {
            this.Turn = turn;
            this.Unit = unit;
            this.ButtonNumber = buttonNumber;
            this.Matrix = matrix;
            this.GameResult = gameResult;
        }

        public TTTpacket()
        {
            Matrix = null;
            GameResult = null;
            Unit = null;
        }
    }
}

File: AHome/10.03/XOClientAndr/app/src/main/java/com/example/ghhos/xoclient/api/TTTPacket.java
package com.example.ghhos.xoclient.api;

/**
 * Created by adm on 27.02.2017.
 */

public class TTTPacket {

    public TTTPacket() {
        Matrix = null;
    }

    public TTTPacket(String playerTurn, String unit, int buttonNumber, String[] matrix, String gameResult) {
        PlayerTurn = playerTurn;
        Unit = unit;
        ButtonNumber = buttonNumber;
        Matrix = matrix;
        GameResult = gameResult;
    }

    public String PlayerTurn;
    public String Unit;
    public int ButtonNumber;
    public String[] Matrix;
    public String GameResult;
}
